### PR TITLE
Review new SBOM page

### PR DESCRIPTION
**Pull request by kprosise**
Review done as a separate commit. A bit of cleanup was done, though no major changes to content required.

Rendered locally, content appeared to be as intended. Link-check failure is due to an external issue.

This relates to doc PR #419

Signed-off-by: Katrina Prosise <katrina.prosise@foundries.io>

## Readiness

* [x] Merge (pending reviews)
* [ ] Merge after _date or event_
* [ ] Draft

## Overview

_Why merge this PR? What does it solve?_

## Checklist

_Optional. Add a 'x' to steps taken._
_You can fill this out after opening the PR. "Did I..."_

* [x] Run spelling and grammar check, preferably with linter.
* [] Avoid changing any header associated with a link/reference.
* [x] Step through instructions (or ask someone to do so).
* [x] Review for [wordiness](https://languagetool.org/insights/post/wordiness/)
* [] Match tone and style of page/section.
* [x] Run `make linkcheck`.
* [x] View HTML in a browser to check rendering.
* [x] Use [semantic newlines](https://bobheadxi.dev/semantic-line-breaks/).
* [x] follow best practices for commits.
  * [x] Descriptive title written in the imperative.
  * [x] Include brief overview of QA steps taken.
  * [x] Mention any related issues numbers.
  * [x] End message with sign off/DCO line (`-s, --signoff`).
  * [x] Sign commit with your gpg key (`-S, --gpg-sign`).
  * [ ] Squash commits if needed.
* [ ] Request PR review by a technical writer and at least one peer.

## Comments

Rather than getting permission to push to @doanac fork, pushed to a new branch of the main repo. @angolini I believe most of your comments were already addressed, but feel free to give feedback.  If all looks good, I can just copy/paste this as a review of the [original pull request](https://github.com/foundriesio/docs/pull/419)

### CODE REVIEW

**Comment by doanac** [comment]
Docs for 94d8184ed6263f565d273ae893939c97d0620b16 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/1692/docs/artifacts/html/index.html

**Review by doanac — APPROVED** [review]
Looks good. I'm not sure how we decide when/where to share web UI views, but we now have a button in the webui to bring up an sbom browser:

image

**Comment by doanac** [comment]
another thing I just thought about - we may want to include a disclaimer stating SBOM generation isn't fool-proof and can miss certain types of packages. I'm checking with legal and will get back to you.

**Comment by kprosise** [comment]
@doanac Yep. I'm somewhat familiar with this sort of issue. I can write up a short statement in the mean time.

**Comment by kprosise** [comment]
As I also posted in slack, perhaps a disclaimer saying something like “Foundries.io does not provide legal advice of license compliance.
It is recommended that SBOM results be verified.
Foundries.io is not liable for any legal repercussions resulting from failure to verify SBOM results and to comply with licensing.”

**Comment by munoz0raul** [comment]
Hi. I just read the docs, looks good but I think we are missing talking about the UI functionalities.

IMHO, what we did in the UI is really really cool, so why not explain that even before fioctl?

image

image

image


@kprosise what do you think about creating a chapter about the UI and mention that we did it to facilitate customers' dally use?

**Comment by kprosise** [comment]
@munoz0raul We have been discussing just that, and I agree, the web UI looks amazing. However there's some decisions that need to be made first, and the plan (to the best of my knowledge) is to merge it as it is, and then add the web UI info in a different commit. I may get started on it now though! Let us chat tomorrow.

File: source/reference-manual/factory/sboms.rst [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+
+A Software Bill of Materials(SBOM) declares *the list of software packages used to build a Target*.
+SBOMs are foundational to understanding:
+
+ * Inventory management — the packages a Target uses.
+ * License compliance — the software licenses of the packages.
+ * Vulnerability management — the package versions.
+
+The FoundriesFactory SBOM feature extracts the SBOM data and analyzes it according to your needs.
+
```
Comment (kprosise) [inline source/reference-manual/factory/sboms.rst:12]:
```suggestion

.. important::
    `Per our terms and conditions <https://foundries.io/company/terms/>`_: FoundriesFactory build SBOMs (“the SBOM data”) are provided for your use and are generated from SPDX metadata in all project source code files. Responsibility for open source license compliance rests with you. In no event shall Foundries.io Limited be liable for any claim, damages or other liability, whether in an action of contract, tort or other legal theory, arising from, out of, or in connection with the use of the SBOM data.

```
I don't know if we want the whole text there, we could just point to it

Comment (kprosise) [reply source/reference-manual/factory/sboms.rst:12]: @doanac thoughts?

Comment (doanac) [reply source/reference-manual/factory/sboms.rst:12]: I think what you have is good. This one thing is a little different than other disclaimers, so it's probably worth highlighting for now.

**Comment by kprosise** [comment]
Added additional reviewers, largely to help keep everyone informed. After the build checks, should we go ahead and merge, adding web UI details later? We could likely squeeze it for this release, especially if the above screenshots still mirror the current state of the web UI.

**Comment by kprosise** [comment]
Link-check  failure can be ignored, as it has already be fixed in main.

**Comment by doanac** [comment]
i don't think the UI has changed or will change. So feel free to include a screenshot if you'd like. Maybe in a new PR since this has grown over time.

**Comment by kprosise** [comment]
@doanac I'll go ahead and merge, and then open up a Jira issue to add web UI details.

**Pull request merged**